libs/sql-ddl/src/postgres.rs

```diff
@@ -486,28 +483,28 @@ mod tests {
     #[test]
     fn full_alter_table_add_foreign_key() {
         let alter_table = AlterTable {
-            table_name: PostgresIdentifier::WithSchema("public".into(), "Cat".into()),
+            table_name: &PostgresIdentifier::WithSchema("public".into(), "Cat".into()),
             clauses: vec![AlterTableClause::AddForeignKey(ForeignKey {
                 constrained_columns: vec!["friendName".into(), "friendTemperament".into()],
                 constraint_name: Some("cat_friend".into()),
                 on_delete: None,
                 on_update: None,
                 referenced_columns: vec!["name".into(), "temperament".into()],
-                referenced_table: "Dog".into(),
+                referenced_table: &"Dog",
             })],
         };
 
         let expected =
-            "ALTER TABLE \"public\".\"Cat\" ADD CONSTRAINT \"cat_friend\" FOREIGN KEY (\"friendName\", \"friendTemperament\") REFERENCES \"Dog\"(\"name\", \"temperament\")";
+            "ALTER TABLE \"public\".\"Cat\" ADD CONSTRAINT \"cat_friend\" FOREIGN KEY (\"friendName\", \"friendTemperament\") REFERENCES Dog(\"name\", \"temperament\")";
```

Comment:
why don't we fully qualify the referenced table?

Reply:
This is at the SQL rendering level — starting with this PR, sql-ddl will only render what we pass in as table name here, and in this test we pass an unquoted string, but in sql-migration-connector, we pass the qualified, quoted table name.

In general I think sql-ddl is a bit of a failed experiment, we should start re-absorbing it into sql-migration-connector.